frontend/src/components/map/CityMap.vue

```diff
@@ -175,9 +176,26 @@ export default class CityMap extends Vue {
   @Prop({ default: undefined })
   private readonly geoJsonOptions?: GeoJSONOptions;
 
-  private layerGroup: LayerGroup = new LayerGroup();
+  /**
+   * Ob die Auswahl von GeoJson deaktiviert werden soll.
+   */
+  @Prop({ type: Boolean, default: false })
+  private readonly geoJsonDisabled!: boolean;
```

Comment:
Wäre es für die zukünftige Verwendung der CityMap nicht besser, das Aktiveren und Deaktiveren von GeoJson zu entfernen und den Marker über die beiden Properties `geoJson` und `geoJsonOptions` zu setzen?

1. CityMap feuert ein Klick-Event.
2. Die darüberliegende Komponenten fängt das Event.
3. Erstellt GeoJson-Punktkoordinate und die Optionen für GeoJson
4. Übergibt die Punktkoordinate samt Options an die CityMap.

Wird die Punktkoordinate durch die Adresse gegeben, wird das GeoJson-Objekt samt den Optionen in der Komponente erstellt welche die CityMap beinhaltet und zur CityMap übergeben.